L1Trigger/L1THGCal/src/backend/HGCalShowerShape.cc

```diff
@@ -532,4 +669,21 @@ void HGCalShowerShape::fillShapes(l1t::HGCalMulticluster& c3d, const HGCalTrigge
   c3d.layer90percent(percentileLayer(c3d, triggerGeometry, 0.90));
   c3d.triggerCells67percent(percentileTriggerCells(c3d, 0.67));
   c3d.triggerCells90percent(percentileTriggerCells(c3d, 0.90));
+  c3d.first1layers(sum_X(c3d, triggerGeometry, 0, 0));
+  c3d.first3layers(sum_X(c3d, triggerGeometry, 2, 0));
+  c3d.first5layers(sum_X(c3d, triggerGeometry, 4, 0));
```

Comment:
- Layer `0` doesn't exist so it is confusing to use it as the start layer
- And a number of layers equal to `0` is even more confusing